### PR TITLE
Update Readme with Limitations

### PR DESCRIPTION
**Pull request by sifex**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -28,6 +28,12 @@ Features
   the fake server, from your webpage
 - **supports webhooks** that you can customize using a special API route
 
+Limitations
+--------
```
Comment (adrienverge) [inline README.rst]: Please add the correct number of `-` and it will be good to go!

Comment (sifex) [reply README.rst]: Ah, like a subheading of features, no problem

Comment (adrienverge) [reply README.rst]:
```
Limitations
-----------
```

Usually in RST, we write the underline a long as the title line itself.

More info at https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Comment (sifex) [reply README.rst]: Ah cool, no worries

**Comment by adrienverge** [comment]
Thanks @sifex!

**Pull request merged**